### PR TITLE
Fix env load. Add DB charset and engine options in settings

### PR DESCRIPTION
**Pull request by radome**

### CODE REVIEW

File: production_services/settings.py [hunk 1]
```diff
@@ -100,7 +100,9 @@
         'HOST': os.getenv('USER_DB_HOST', '127.0.0.1'),
         'PORT': os.getenv('USER_DB_PORT', '3306'),
         'OPTIONS': {
-            "init_command": "SET default_storage_engine=MYISAM",
+            "init_command": "SET default_storage_engine={}".format(
```
Comment (marcoooo) [inline production_services/settings.py]: Actually I just removed completely this statement in another PR :-)

Comment (radome) [reply production_services/settings.py]: Okay, the parametrised charset could be useful though. Should I remove the engine line?

Comment (marcoooo) [reply production_services/settings.py]: Since this is only used when creating the database schema (hence only for automated tests) I would say yes.

Comment (radome) [reply production_services/settings.py]: Reverted in ead117e

**Pull request merged**